resources/scripts.js

```diff
@@ -169,6 +169,27 @@ function gameOver() {
     betButtonState(false);
 }
 
+async function playerMoneyChange(change) {
+    playerMoneyElement.innerHTML = playerMoney;
+    if (change === 'subtract') {
```

Comment:
Possibly think of changing this to either comparing `change` to a defined constant such as `const SUBTRACT = 'subtract'` OR do a simple math comparison of the user's previous and current money count

Reply:
Will create a new variable saving the last state of playerMoney. This will be used as a comparison.